packages/flutter/lib/src/material/choice_chip.dart

```diff
@@ -148,7 +148,7 @@ class ChoiceChip extends StatelessWidget
     assert(debugCheckHasMaterial(context));
     final ChipThemeData chipTheme = ChipTheme.of(context);
     final ChipThemeData? defaults = Theme.of(context).useMaterial3
-      ? _FilterChipDefaultsM3(context, isEnabled, selected)
+      ? _ChoiceChipDefaultsM3(context, isEnabled, selected)
```

Comment:
Thanks for doing this, I just caught this myself working on another Chip issue. Not sure how this got messed up.

Reply:
No problem!